src/js/import.js

```diff
@@ -241,6 +233,31 @@
     });
   }
 
+  function addClickableButtonEventListener(button, CSVHeaders){
+    button.click(function() {
+      if(button.is(":checked")){
+        var potentialComplaint = button.parent().nextAll();
+        potentialComplaint.remove();
```

Comment:
Do you use `potentialComplaint` in the function somewhere? Otherwise:
```js
// Remove all potential complaints
button.parent().nextAll().remove();
```